benchmarks/fibonacci/docker-compose.yaml

```diff
@@ -0,0 +1,81 @@
+version: "3.9"
+services:
+  ## Python
+  fibonacci-python:
+    image: davidschall/fibonacci-python:v1
+    container_name: fibonacci-python
+    build: 
+      context: ./
+      dockerfile: ./python/Dockerfile.python
+    ports:
+    - published: 50061
+      target: 50051
+
+  # fibonacci-cython:
+  #   image: davidschall/fibonacci-cython:v1
+  #   container_name: helloworld-cython
+  #   build: 
+  #     context: ./
+  #     dockerfile: ./python/Dockerfile.cython
+  #   ports:
+  #   - published: 50062
+  #     target: 50051
+
+  fibonacci-pypy:
```

Comment:
Only use python for now. The other runtimes we can maybe add later but its already enough